### DISCUSSION TITLE
Language

### DISCUSSION THREAD

**Discussion by Raquezin** [dsdanielpark/Bard-API#218] [Ideas]
Please you can update ALLOWED_LANGUAGES in bard_constants.py

**Comment by dsdanielpark** [comment]
It has been updated. However, there may be differences in the mapping for standard languages, so some languages may encounter errors.

The update is complete, but QA and development for other extensions are still ongoing, so the official update to PyPI will be done later, probably in version 0.1.39 or 0.1.40. Until then, please install and use the GitHub version.

Thank you.

```
$ pip install git+https://github.com/dsdanielpark/Bard-API.git
```

**Reply by Raquezin** [reply]
Thank you I'm going to update with git clone
